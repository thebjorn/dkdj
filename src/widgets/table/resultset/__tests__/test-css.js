
import $ from 'jquery';


test("test-flex-1-css", () => {
    document.body.innerHTML = `
        <div id="work">
            <div id="a"></div>
        </div>
    `;
    const work = $('#work');

    $('#a').css({flex: 1});

    console.log(work.html());

    expect(document.getElementById('work')).toMatchSnapshot();
});
