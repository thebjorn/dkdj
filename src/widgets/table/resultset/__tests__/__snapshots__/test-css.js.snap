// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test-flex-1-css 1`] = `
<div
  id="work"
>
  
            
  <div
    id="a"
  />
  
        
</div>
`;
